libr/arch/p/arm/armass64.c

```diff
@@ -1,5 +1,9 @@
 /* radare - LGPL - Copyright 2015-2023 - pancake */
 
+#include "r_types.h"
```

Comment:
all thse includes are already in by r_util.h

Reply:
removed in latest commit